<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TQtSettings</class>
 <widget class="QDialog" name="TQtSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="itpploganalyzer.qrc">
    <normaloff>:/resources/itpploganalyzer.png</normaloff>:/resources/itpploganalyzer.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageCharacter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>477</width>
        <height>496</height>
       </rect>
      </property>
      <attribute name="label">
       <string>File indicators</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>491</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayoutFileSettings">
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditInfo">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the tag marking a line as information&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>INF</string>
          </property>
          <property name="maxLength">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="labelColAlign">
          <property name="text">
           <string>Column alignment</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelEnd">
          <property name="text">
           <string>End indicator of block</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelDebug">
          <property name="text">
           <string>Debug tag</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QLabel" name="labelColorWarning">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelInfo">
          <property name="text">
           <string>Info tag</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelError">
          <property name="text">
           <string>Error tag</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEditError">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the tag marking a line as error&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>ERR</string>
          </property>
          <property name="maxLength">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEditTrace">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the tag marking a line as debug&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>TRC</string>
          </property>
          <property name="maxLength">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QToolButton" name="toolButtonColTrace">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="itpploganalyzer.qrc">
            <normaloff>:/resources/color-wheel_256x256.png</normaloff>:/resources/color-wheel_256x256.png</iconset>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QToolButton" name="toolButtonColError">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="itpploganalyzer.qrc">
            <normaloff>:/resources/color-wheel_256x256.png</normaloff>:/resources/color-wheel_256x256.png</iconset>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="4">
         <widget class="QLabel" name="labelColorDebug">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="5">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="13" column="1" colspan="4">
         <widget class="QLineEdit" name="lineEditDelimeter">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Einter the delimiter used to separate the columns.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>,</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QLabel" name="labelColorTrace">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="labelColumns">
          <property name="text">
           <string>Number columns</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QToolButton" name="toolButtonColDebug">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="itpploganalyzer.qrc">
            <normaloff>:/resources/color-wheel_256x256.png</normaloff>:/resources/color-wheel_256x256.png</iconset>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="labelDelimeter">
          <property name="text">
           <string>Column delimeter</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEditDebug">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the tag marking a line as debug&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>DBG</string>
          </property>
          <property name="maxLength">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="labelColorInfo">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditWarning">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the tag marking a line as warning&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>WRN</string>
          </property>
          <property name="maxLength">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QLabel" name="labelColorError">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelWarning">
          <property name="text">
           <string>Warning tag</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1" colspan="4">
         <widget class="QLineEdit" name="lineEditColAlign">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Elnter the alignment for each column; &lt;b&gt;l&lt;/b&gt;=left, &lt;b&gt;r&lt;/b&gt;=right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>l,r,l,l,l,l,r,l</string>
          </property>
          <property name="maxLength">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="12" column="4">
         <widget class="QListWidget" name="listWidgetColumns">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the titles for each column&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="layoutMode">
           <enum>QListView::LayoutMode::SinglePass</enum>
          </property>
          <item>
           <property name="text">
            <string>Timestamp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>User</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Type</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SType</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>File</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Content</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QToolButton" name="toolButtonColInfo">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="itpploganalyzer.qrc">
            <normaloff>:/resources/color-wheel_256x256.png</normaloff>:/resources/color-wheel_256x256.png</iconset>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="labelTrace">
          <property name="text">
           <string>Trace tag</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelStart">
          <property name="text">
           <string>Start indicator of block</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QToolButton" name="toolButtonColWarning">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="itpploganalyzer.qrc">
            <normaloff>:/resources/color-wheel_256x256.png</normaloff>:/resources/color-wheel_256x256.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="4">
         <widget class="QLineEdit" name="lineEditEnd">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the end indicator of a block&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>}exit:</string>
          </property>
          <property name="maxLength">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QLineEdit" name="lineEditStart">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the start indicator of a block&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>{entry:</string>
          </property>
          <property name="maxLength">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="labelThreadID">
          <property name="text">
           <string>Column with Tread ID</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QSpinBox" name="spinBoxColumns">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the number of columns the log file has&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QToolButton" name="toolButtonValue">
          <property name="toolTip">
           <string>Enter JSON value names</string>
          </property>
          <property name="whatsThis">
           <string>Button to enter JSON value names.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="itpploganalyzer.qrc">
            <normaloff>:/resources/json_64x64.png</normaloff>:/resources/json_64x64.png</iconset>
          </property>
         </widget>
        </item>
        <item row="15" column="1" colspan="4">
         <widget class="QSpinBox" name="spinBoxThreadID">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the column containing the &lt;i&gt;thread ID&lt;/i&gt;, if there is any.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="pageElse">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>477</width>
        <height>496</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Other settings</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>491</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayoutOtherSettings">
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelLogfile">
          <property name="text">
           <string>Log file (path+name)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QToolButton" name="toolButtonSourcePath">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelResultPath">
          <property name="text">
           <string>Result path</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="toolButtonResultPath">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditLogfile">
          <property name="text">
           <string>itpploganalyzer.log</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelSourcePath">
          <property name="text">
           <string>Default soure directory</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEditSourcePath">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelLogLevel">
          <property name="text">
           <string>Log level</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="toolButtonLogfile">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEditResultPath">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="spinBoxLogLevel">
          <property name="maximum">
           <number>6</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditStart</tabstop>
  <tabstop>lineEditEnd</tabstop>
  <tabstop>lineEditInfo</tabstop>
  <tabstop>toolButtonColInfo</tabstop>
  <tabstop>lineEditWarning</tabstop>
  <tabstop>toolButtonColWarning</tabstop>
  <tabstop>lineEditError</tabstop>
  <tabstop>toolButtonColError</tabstop>
  <tabstop>lineEditDebug</tabstop>
  <tabstop>toolButtonColDebug</tabstop>
  <tabstop>lineEditTrace</tabstop>
  <tabstop>toolButtonColTrace</tabstop>
  <tabstop>spinBoxColumns</tabstop>
  <tabstop>toolButtonValue</tabstop>
  <tabstop>listWidgetColumns</tabstop>
  <tabstop>lineEditDelimeter</tabstop>
  <tabstop>lineEditColAlign</tabstop>
  <tabstop>spinBoxThreadID</tabstop>
  <tabstop>lineEditLogfile</tabstop>
  <tabstop>lineEditResultPath</tabstop>
  <tabstop>toolButtonSourcePath</tabstop>
  <tabstop>toolButtonResultPath</tabstop>
  <tabstop>toolButtonLogfile</tabstop>
  <tabstop>spinBoxLogLevel</tabstop>
  <tabstop>lineEditSourcePath</tabstop>
 </tabstops>
 <resources>
  <include location="itpploganalyzer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TQtSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TQtSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
